lib/WebpackError.js

```diff
@@ -0,0 +1,21 @@
+/*
+	MIT License http://www.opensource.org/licenses/mit-license.php
+	Author Jarid Margolin @jaridmargolin
+*/
+"use strict";
+
+module.exports = class WebpackError extends Error {
+	constructor() {
+		super();
+
+		if(this.initialize) {
+			this.initialize.apply(this, arguments);
```

Comment:
@sokra - I am willing to do the requested work, but wanted to quickly elaborate why this change was made. Normalizing Error class creation has benefits. Multiple issues existed due to each Error subclass implementing constructor differently.

**Issues where stacktrace was  added prior setting name and message properties:** 

https://github.com/webpack/webpack/commit/7c09a4d95295b13abdd2f1688f3729faf1e4c051
https://github.com/webpack/webpack/commit/18c313097a96212ad866ba6e7e7323e4736afca9

**Host of different mechanisms for calling the capture stack trace method**

Should it be wrapped in an if block? Should it pass this.constructor, or should it hardcode the parent class? etc...

https://github.com/jaridmargolin/webpack/blob/master/lib/ModuleParseError.js#L26
https://github.com/jaridmargolin/webpack/blob/master/lib/CaseSensitiveModulesWarning.js#L18

---

This initialize method is never intended to be used anywhere besides bootstrapping the instance. I could see how a pattern like this could be abused, but do you think this specific use case is an abuse?